raiden/tests/utils/transport.py

```diff
@@ -215,7 +215,7 @@ def matrix_server_starter(
                         '-m',
                         'synapse.app.homeserver',
                         f'--server-name={server_name}',
-                        f'--config-path={config_file.name}',
+                        f'--config-path={config_file!s}',
```

Comment:
`!s` is the default coercion. So it should even work with just the plain variable.

Reply:
I did it this way as i noticed it's done somewhere else like this so i opted for consistency. I am ok with this both ways.